andela/travel_tool_front
# #161759224 Rename instances of the word delete to disable

#### What does this PR do?
- Renames both instances having the word delete to disable on the travel checklist page

#### Description of Task to be completed?
- Rename both instances having the word Delete to Disable on the Travel checklist page
Both instances are:
  - Delete button
  - Deleted Items section

#### How should this be manually tested?
- Setting up the backend
     - Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
     - Enter into the project directory - `cd travel_tool_back`
     - checkout the branch git checkout `develop`
     - Install dependencies using `yarn install`
     - Rollback and run database migrations with `yarn db:rollmigrate`


- Setting up the frontend
   - Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
   - Enter into the project directory - `cd travel_tool_front`
   - Checkout to the branch `ch-rename-delete-disable-161759224` then run the app
   - login to the application
   - navigate to the trip planner page as a travel admin
   - click the disable button on a checklist it and verify that the word disable has replaced delete.
   - make sure the modal matches the [implementation](https://projects.invisionapp.com/d/main#/console/14861108/324508456/preview)

#### Any background context you want to provide?
- The errors flagged by codeClimate are prop-type validation errors which are for prop types that have similar structures but are not the same in any way

#### What are the relevant pivotal tracker stories?
[#161759224](https://www.pivotaltracker.com/story/show/161759224)

#### Screenshots (if appropriate)
<img width="507" alt="screen shot 2018-11-09 at 21 32 16" src="https://user-images.githubusercontent.com/21138053/48281250-0e734580-e467-11e8-8ee1-d847b754c9fb.png">
